kalite/distributed/static/js/distributed/user/views.js

```diff
@@ -314,4 +314,64 @@ window.UserView = BaseView.extend({
     logout: function() {
         this.model.logout();
     }
-});
\ No newline at end of file
+});
+
+/* This view toggles which navbar items are displayed to each type of user */ 
+window.ToggleNavbarView = BaseView.extend ({
+
+    template: HB.template("user/navigation"),
+
+    initialize: function() {
+
+        _.bindAll(this);        
+        this.listenTo(this.model, "change", this.render);
+        $("topnav").append(this.template());
+
+    },
+
+    render: function() {
+        
+        this.facility_id = {"facility_id": this.model.get("facility_id")};
+        this.$el.html(this.template(this.facility_id));
+        
+        this.userView = new UserView({ model: this.model, el: "#topnav" });
+        $("#topnav").prepend(new AutoCompleteView().$el);
+        
+        // GUEST
```

Comment:
Easiest thing to do is just to pass window.statusModel.attributes directly to the template (and use the model's property names in the template).

window.statusModel.attributes gives you a Javascript object, just like you are defining with this.user_info.